src/codegen/query_compiler.cpp

```diff
@@ -119,7 +119,7 @@ bool QueryCompiler::IsExpressionSupported(
     const expression::AbstractExpression &expr) {
   switch (expr.GetExpressionType()) {
     case ExpressionType::STAR:
-    case ExpressionType::FUNCTION:
+    // case ExpressionType::FUNCTION:
```

Comment:
Remove this. We'll fix when we have function parity with existing value-based functions.